database/migrations/2018_01_02_065902_create_posts_table.php

```diff
@@ -20,9 +20,14 @@ public function up()
                   ->references('id')->on('users')
                   ->onUpdate('cascade')
                   ->onDelete('cascade');
-            $table->integer('postable_id')->unsigned();
-            $table->string('postable_type', 30);
+            $table->integer('book_id')->unsigned();
+            $table->foreign('book_id')
+                  ->references('id')->on('books')
+                  ->onUpdate('cascade')
+                  ->onDelete('cascade');
+            $table->string('type', 30)->comment="review || find || status";
```

Comment:
- you should use int(1) to set type for column.

Reply:
- I think type of `type` is string. because we have 3 id and type of their is int. Avoid mistake when query